custom/icds/views/custom_sms_report.py

```diff
@@ -8,23 +8,35 @@
 from custom.icds.forms import CustomSMSReportRequestForm
 from custom.icds.tasks.sms import send_custom_sms_report
 from custom.icds_core.const import SMSUsageReport_urlname
+from custom.icds.utils.custom_sms_report import CustomSMSReportTracker
 
 
 @method_decorator(toggles.ICDS_CUSTOM_SMS_REPORT.required_decorator(), name='dispatch')
 class SMSUsageReport(BaseMessagingSectionView):
     template_name = 'icds/sms/custom_sms_report.html'
     urlname = SMSUsageReport_urlname
     page_title = _('Custom SMS Usage Report')
+    report_tracker = CustomSMSReportTracker()
```

Comment:
Can this be on the class? You are assigning an instance and that will get bound to the class itself.